beacon-chain/p2p/subnets.go

```diff
@@ -134,9 +201,28 @@ func attSubnets(record *enr.Record) ([]uint64, error) {
 	return committeeIdxs, nil
 }
 
+// Reads the sync subnets entry from a node's ENR and determines
+// the committee indices of the sync subnets the node is subscribed to.
+func syncSubnets(record *enr.Record) ([]uint64, error) {
+	bitV, err := syncBitvector(record)
```

Comment:
You used `bitS` earlier, perhaps use `bitS` for sync related

Reply:
Same reason as mentioned above.